README.md

```diff
@@ -358,8 +357,13 @@ All [configuration](#configuration) is done via config.ini. Examples are comment
 
 Please open a new issue on github, only here we can work on your problem in a structured way: <https://github.com/henne49/dbus-opendtu/issues/new/choose>
 
-⚠️ **Change the Logging Parameter under DEFAULT in /data/dbus-opendtu/config.ini to Logging = DEBUG, please revert back to ERROR or CRITICAL, once debugging and troubleshooting is complete. Rerun the script and share the current log file in /var/log/dbus-opendtu/current**.
+⚠️ **Change the Logging Parameter under DEFAULT in /data/dbus-opendtu/config.ini to Logging = DEBUG, please revert back to ERROR or CRITICAL, once debugging and troubleshooting is complete.**
 
+Rerun the script and share the log file current.log use the following command to generate human readable timestamps. We require this to start any troubleshooting.
+
+```
+cat /var/log/dbus-opendtu/current | tai64nlocal > current.log
```

Comment:
I suggest to name the readable file differently here. This makes communication easier e.g. when asking for the current.readable.log